### DISCUSSION TITLE
Who was assassinated at the Lorraine Motel in Memphis, TN?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#304] [Galaxy Trivia] [answered]
**Hint:** MLK

**Comment by thuongtruong1009** [comment, marked as answer]
Civil rights leader and activist Martin Luther King Jr. was assassinated at the Lorraine Motel in Memphis, TN on April 4, 1968.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [🥇 Galaxy Brain](https://github.com/thuongtruong1009?tab=achievements&achievement=galaxy-brain)!
Continue to unlock other badge levels (*how about* 🥈 *Pair Extraordinaire*?), and don't hesitate to post your own questions to the community. 😊🎉

**Reply by thuongtruong1009** [reply]
thank you very much 🥰❤️ @ParanoidUser

**Comment by rubikproxy** [comment]
Dr. Martin Luther King Jr., the prominent civil rights leader, was assassinated at the Lorraine Motel in Memphis, Tennessee, on April 4, 1968. His assassination was a pivotal moment in American history, sparking widespread mourning and accelerating the civil rights movement.

**Comment by nguyenhoanghai1605** [comment]
Hi @ParanoidUser,
Martin Luther King Jr. was assassinated at the Lorraine Motel in Memphis, Tennessee, on April 4, 1968. He was shot while standing on the balcony outside his second-floor room.
